sources/oracle/data_test.go

```diff
@@ -201,31 +143,31 @@ func TestConvertMultiColData(t *testing.T) {
 			cols:   []string{"a", "b", "c"},
 			colIds: []string{"c1", "c2", "c3"},
 			vals:   []string{"6", "6.6", "1"},
-			ecols:  []string{"a", "b", "c"},
+			ecols:  []string{"c1", "c2", "c3"},
```

Comment:
Has the definition of the test generator changed to the take the column id's instead of the values ?